modules/sdk-coin-atom/test/unit/atom.ts

```diff
@@ -209,6 +232,31 @@ describe('ATOM', function () {
       });
     });
 
+    it('should explain sendMany transfer transaction', async function () {
```

Comment:
can we add one testcase for sendMany delegate and undelegate transfer type also.

Reply:
same for any abstract-cosmos coin also.